multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -547,6 +548,15 @@ def general_stats_headers (self):
         'shared_key': 'read_count',
         'hidden': True
     }
+    self.general_stats_headers['general_error_rate'] = {
+        'title': 'Error rate'.format(config.read_count_prefix),
+        'description': 'General error rate of the reads',
```

Comment:
Could you write a slightly longer description here please? eg. Specify that it's the alignment error rate, and perhaps how it's calculated or something?